app/upgrades/v14/upgrades.go

```diff
@@ -0,0 +1,59 @@
+package v14
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/CosmosContracts/juno/v14/app/keepers"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	globalfeetypes "github.com/cosmos/gaia/v9/x/globalfee/types"
+)
+
+// Returns "ujunox" if the chain is uni, else returns the standard ujuno token denom.
+func GetChainsDenomToken(chainID string) string {
+	if strings.HasPrefix(chainID, "uni-") {
+		return "ujunox"
+	}
+	return "ujuno"
+}
+
+func CreateV14UpgradeHandler(
+	mm *module.Manager,
+	cfg module.Configurator,
+	keepers *keepers.AppKeepers,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		logger := ctx.Logger().With("upgrade", UpgradeName)
+
+		nativeDenom := GetChainsDenomToken(ctx.ChainID())
+		logger.Info(fmt.Sprintf("With native denom %s", nativeDenom))
+
+		logger.Info(fmt.Sprintf("pre migrate version map: %v", vm))
+
+		// Run migrations
+		versionMap, err := mm.RunMigrations(ctx, cfg, vm)
+
+		logger.Info(fmt.Sprintf("post migrate version map: %v", versionMap))
+
+		// GlobalFee
+		minGasPrices := sdk.DecCoins{
+			// 0.0025ujuno
+			sdk.NewDecCoinFromDec(nativeDenom, sdk.NewDecWithPrec(25, 4)),
+			// 0.001 ATOM CHANNEL-1 -> `junod q ibc-transfer denom-trace ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9`
```

Comment:
at 0.001 atom per gas unit, that cost becomes $0.01236 at current ATOM price of $12.36

For juno, its $1.01*0.0025 = $0.002525 per gas. Or roughly 5x cheaper at current market rates

Maybe we propose in the future to lower ATOM gas cost? or is it meant to be a significant premium? Does anyone even use. If not, we should lower to get more adoption for a more diverse fee income pool